<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NgiiDataUtilsDockWidgetBase</class>
 <widget class="QDockWidget" name="NgiiDataUtilsDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>국토지리정보원 데이터 중첩검사</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="btnLoadVector">
      <property name="text">
       <string>온맵/지형도/공공측량성과 불러오기</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="btnLoadImage">
      <property name="text">
       <string>영상(GeoTIFF/IMG) 불러오기</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btnLoadTms">
      <property name="text">
       <string>국가 인터넷지도 불러오기</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btnLoadWms">
      <property name="text">
       <string>최신 국토기본정보 불러오기</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QProgressBar" name="prgSub">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>3</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>3</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QProgressBar" name="prgMain">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>5</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblStatus">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>진행상황 표시</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>213</width>
         <height>136</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>130</height>
           </size>
          </property>
          <property name="title">
           <string>[온맵](B090)온맵_37612068.pdf</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horLayout1_1">
             <item>
              <widget class="QPushButton" name="btnToSpWin_1">
               <property name="text">
                <string>분할창으로 띄우기</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemove_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>제거</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horLayout2_1">
             <item>
              <widget class="QLabel" name="lblTrans_1">
               <property name="text">
                <string>투명도:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Phonon::SeekSlider" name="sldTrans_1"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horLayout3_1">
             <item>
              <widget class="QLabel" name="lblColor_1">
               <property name="text">
                <string>색  상:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSelColor_1">
               <property name="text">
                <string>COLOR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnResetColor_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>초기화</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder>btnToSpWin_1</zorder>
          <zorder>lblTrans_1</zorder>
          <zorder>sldTrans_1</zorder>
          <zorder>toolButton</zorder>
          <zorder>lblColor_1</zorder>
          <zorder>btnSelColor_1</zorder>
          <zorder>btnResetColor_1</zorder>
          <zorder>btnRemove_1</zorder>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>117</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
